scripts/test-runner.js

```diff
@@ -20,6 +20,7 @@ const DEV_CLUSTER_CONFIG = `
         xsi:schemaLocation="http://www.hazelcast.com/schema/config
         http://www.hazelcast.com/schema/config/hazelcast-config-4.0.xsd">
         <cluster-name>dev</cluster-name>
+        <jet enabled="true"></jet>
```

Comment:
some code samples use sql, so we need to enable it in the cluster that code samples are run against